app/models/user.rb

```diff
@@ -2,14 +2,6 @@ class User < ApplicationRecord
   validates_presence_of :name, :address, :city, :state, :zip
   validates :email, uniqueness: true, presence: true
   validates_presence_of :password_digest, require: true
-
   has_secure_password
-
-  def default?
-    role == 0
-  end
-
-  def merchant?
-    role == 1
-  end
+  enum role: %w(default merchant admin)
```

Comment:
love the enum. can `default` be `user`?

Reply:
I'm not married to default so whatever works really!

Reply:
Came to a decision on default over user.